rllib/algorithms/cql/cql.py

```diff
@@ -150,6 +197,12 @@ def validate(self) -> None:
             )
             try_import_tfp(error=True)
 
+        # Assert that for a local learner the number of iterations is 1. Note,
```

Comment:
nit: We shouldn't change values under the hood and instead simply error here with a descriptive message for the user (exactly what you wrote in the comment :) ).